bindgen/gen_odin.py

```diff
@@ -425,6 +427,13 @@ def gen_c_imports(inp, c_prefix, prefix):
             args = funcdecl_args_c(decl, prefix)
             res_type = funcdecl_result_c(decl, prefix)
             res_str = '' if res_type == '' else f'-> {res_type}'
+            if decl.get('comment'):
+                if '\n' in decl["comment"]:
+                    l("    /*")
+                    l("    " + "    ".join(decl['comment'].splitlines(True)))
+                    l("    */")
+                else:
+                    l("    // " + decl['comment'].strip())
```

Comment:
Might be worth trying to move those three similar code blocks to code-generate the comments into a helper function (which then also covers the slight differences between those three code blocks).

Especially because new language code generators (or updates like this) are often copy-pasted from an existing generator.